vsg/__main__.py

```diff
@@ -33,6 +33,8 @@ def parse_command_line_arguments():
     parser.add_argument('-b', '--backup', default=False, action='store_true', help='Creates a copy of input file for comparison with fixed version.')
     parser.add_argument('-oc', '--output_configuration', default=None, action='store', help='Write configuration to file name.')
     parser.add_argument('-rc', '--rule_configuration', default=None, action='store', help='Display configuration of a rule')
+    parser.add_argument('--style', action='store', default='base', choices=get_predefined_styles(), help='Use predefined style')
+#    parser.add_argument('--skip_phase', nargs='+', default=None, help='Skips entire phases of analysis')
```

Comment:
What about this line? Shoud it be removed or uncommented?

Reply:
I removed it.

Reply:
Update has been pushed.